configmaptocrs/README.md

```diff
@@ -92,3 +92,43 @@ status: {}
 ---
 
 ```
+
+## Run with ConfigMap Volume
```

Comment:
I'd rename this "Running directly against a cluster" and remove the mention of the issue (or add to the bottom).

Reply:
done